code/__DEFINES/math.dm

```diff
@@ -49,9 +49,8 @@
 ///
 /// * This is not replaced by native ceil(), as that is always CEILING(x, 1)!
 #define CEILING(x, y) ( -round(-(x) / (y)) * (y) )
-/// x to the nearest lower multiple of y
-///
-/// * This is not replaced by native floor(), as that is always FLOOR(x, 1)!
+
+/// round() acts like floor(x, 1) by default but can't handle other values
```

Comment:
old documentation still applies doesn't it

Reply:
newdoc still says that it is just `floor(x, 1)`